packages/kbn-test/src/functional_tests/lib/run_elasticsearch.ts

```diff
@@ -10,6 +10,7 @@ import { resolve } from 'path';
 import type { ToolingLog } from '@kbn/tooling-log';
 import getPort from 'get-port';
 import { REPO_ROOT } from '@kbn/repo-info';
+import { ArtifactLicense } from '@kbn/es/src/artifact';
```

Comment:
If it's possible, I think try re-exporting the type through an already existing entry point. It's often a code-stink to see direct imports from other packages' src folders.